### PR TITLE
Added password reset tutorial to FAQ

### LINKED ISSUE #841: Add password reset tutorial to FAQ
## Documentation request
Document password reset procedure on our documentation framework

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do? 
Fixes #841 
Adds password reset tutorial to FAQ

image

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [x] Is the TravisCI build passing?

### CODE REVIEW

File: docs/content/FAQ/_index.md [hunk 1]
```diff
@@ -35,6 +36,23 @@ If you want to see what has changed between versions, refer to the [releases pag
...
+
+On your first access of Monkey Island server, you'll be prompted to create an account. If you forgot the credentials you
+ entered or just want to change them, you need to manually alter the `server_config.json` file. On Linux, this file is 
+ located on `/var/monkey/monkey_island/cc/server_config.json`. On windows, it's based on your install directory (typically 
+ `C:\Program Files\Guardicore\Monkey Island\monkey_island\cc\server_config.json`). Reset the contents of this file back to
+
+```json
+{
+  "server_config": "password",
+  "deployment": "windows"
```
Comment (acepace) [inline docs/content/FAQ/_index.md:49]: Windows or Linux? What happens if someone inserts the wrong one?

File: docs/content/FAQ/_index.md [hunk 2]
```diff
@@ -35,6 +36,23 @@ If you want to see what has changed between versions, refer to the [releases pag
...
+}
+```
```
Comment (acepace) [inline docs/content/FAQ/_index.md:51]: Would rather not give the example with a deployment option that may be incorrect.
Better to tell them what keys to delete.

Comment (acepace) [reply docs/content/FAQ/_index.md:51]: ?

Comment (VakarisZ) [reply docs/content/FAQ/_index.md:52]: Addressed this by specifying in bold, to leave deployment option unchanged. I don't think reading a list of keys (which we'll have to update) is easier than just copy pasting the code and then copy pasting the previous deployment.

Comment (acepace) [reply docs/content/FAQ/_index.md:52]: Agreeing

**Review by acepace — APPROVED** [review]
(づ￣ 3￣)づ

**Pull request merged**